README.md

```diff
@@ -83,6 +83,8 @@ We use feature flags to optionally enable/disable functionality for the Gateway
 * WEBHOOK_TLS_JKS_FILE_LOCATION: If you are enabling TLS as per the above mentioned feature flag and do not wish to use the unsafe bundled keystore, you can provide a location to load your own keystore from using this env var. This location should be a file path available at startup for the service.
 * WEBHOOK_TLS_JKS_PASSWORD: If you are using your own Keystore as per the above env var, then you need to provide a Base64 encoded password to access the keystore via this env var.
 
+* VERSIONED_HOSTS_ENABLED: When this is true, our stackset feature will generate a version-specific host with all auth rules intact for accessing a specific version of a service. 
```

Comment:
Should I have this configured per crd, or leave it per cluster as it is now?

Reply:
I'm always in favour of not adding more bloat into the CRD until somebody actually requests it. Or YAGNI as @cjbooms likes to say...